setup.py

```diff
@@ -22,7 +22,7 @@
 install_requires = [
     "pip>=7",
     "packaging>=23.2",
-    "importlib_resources;python_version>'3.8' and python_version<'3.12'",
```

Comment:
Why did you remove the Python version specifier? FWICS the code is still unchanged, and built-in `importlib.resources.files()` are used on Python 3.12.

Reply:
I removed the constraint smaller than Python 3.12 because the code is working for this Python version. 
Did you find and issue with this version?

Reply:
The code is not used with this  version at all. This change introduces an unnecessary dependency on package that isn't used, and that distributions such as Gentoo don't provide for py3.12 at all (because they're never necessary).

Reply:
@mgorny has a good catch actually, thanks for spotting. Let's try to make a quick patch to undo this part ➕

Reply:
resolving this here: https://github.com/pypiserver/pypiserver/pull/627